components/Banner.tsx

```diff
@@ -30,19 +30,21 @@ const Banner = ({ bannerHeight, setBannerHeight }: {
 					return `h-${bannerHeight}`;
 				})
 				break;
-			case "incompatible-browser":
-				setBannerHeight(() => {
-					const bannerHeight = 24;
-					setBannerHTML((<>
-						<Image src={chromeLogo} alt="chrome extension logo" className={`inline m-6 mr-8 border border-white rounded-full w-${bannerHeight - 12}`}></Image>
-						<p className='text-center font-bold text-xl w-fit sm:max-w-1/2 h-fit my-auto'>
-							Vibinex is only supported in Chromium browsers<br />
-							<span className='text-sm font-normal'>Google Chrome, Microsoft Edge, Opera, Chromium, Brave etc.</span>
-						</p>
-					</>))
-					return `h-${bannerHeight}`;
-				})
-				break;
+            case "incompatible-browser":
+                setBannerHeight(() => {
+                    const bannerHeight = 24;
+                    setBannerHTML((<>
+                        <div className="flex items-center m-6">
```

Comment:
Why create another `div`? For `items-center`?
I strongly recommend against this.

(And even if you really really want to do this, you should have replaced the `React.Fragment`, i.e. the empty `<>` tag, with div instead of nesting another div.)